app/views/users/new.html.erb

```diff
@@ -1,2 +1,7 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<h1>Create your account</h1>
```

Comment:
Use translations in the views

Reply:
Will add this in a seperate feature branch 👍